main.tf

```diff
@@ -49,7 +51,8 @@ resource "observe_dataset" "ingress_logs" {
           podName,
           containerName,
           remainder,
-          log
+          log,
+          request
```

Comment:
I'm keeping the field for backward compatibility. I'm not sure if anybody is using column "request" though